fm/models/spotify.py

```diff
@@ -8,11 +8,17 @@
 Models for storing Spotify Track data.
```

Comment:
Yes I totally agree it is becoming a little misleading now, would be best named as `models/player.py` or something similar.